andela/ah-shakas-frontend
# #161837243 redirect user after verification

## What does this PR do?
It implements redirect to login page after verification.

## Description of Task to be completed?
Once a user has verified their account from their email, they should be redirected back to the application to a page where they are greeted with a message telling them that they have successfully activated their account.

## How should this be manually tested?
Clone this branch ft-account-verification-161837243
Run the following commands on your terminal:
`yarn install` - to install required dependencies
`yarn start` - to start server
When the server starts, you will be redirected to the browser
Navigate to this url: [http://localhost:3000/signup](http://localhost:3000/signup) and enter details to create an account.
After successful signup as shown below; check your email for a verification email sent with a link containing a token to verify your account.

![image](https://user-images.githubusercontent.com/34505753/48490576-c96c5c00-e836-11e8-8fe2-333d83c502c5.png)

**Screenshot of email verification link:**  Click on the link in the email, this should take you to the login page.

![image](https://user-images.githubusercontent.com/34505753/48490649-edc83880-e836-11e8-959a-d91dad561211.png)

## Any background context you want to provide?
N/A

## What are the relevant pivotal tracker stories?
[161837243](https://www.pivotaltracker.com/n/projects/2198533/stories/161837243)

## Deployment note
N/A

## Related PRs branch | PR (branch_name) | (pr_link)
[ft-account-verification-161837243](ft-account-verification-161837243)

## Todos
